### ISSUE TITLE
What's new / "recent_changes" returns None

### ISSUE THREAD

**Issue by ayush-mandowara-bst** [danieliu/play-scraper#32]
**Describe the bug**
What's New / "recent_changes" returns None even though it exists.

**To Reproduce**
```
import play_scraper as ps

data = ps.details('com.supercell.clashofclans')
print(data['recent_changes'])
```

**Expected behavior**
Should return the value for 'what's new' when it exists.

**Screenshots**
Not required 

**Desktop (please complete the following information):**
 - OS: [Windows 10]
 - Python Version [3.7.2]
 - play_scraper Version [0.5.2]

**Additional context**
Issue can be resolved by changing code to 
`recent_changes = changes_soup.text
`

**Comment by ayush-mandowara-bst** [comment]
try:
        changes_soup = soup.select('div[itemprop="description"] content')[1]
        recent_changes = '\n'.join([x.string.strip() for x in changes_soup])
    except (IndexError, AttributeError):
        recent_changes = None
^ from utils.py
It fails for many apps for example: com.supercell.clashofclans & com.instagram.zara

Fix: Change 
`recent_changes = '\n'.join([x.string.strip() for x in changes_soup])`
to
`recent_changes = changes_soup.text`

**Comment by danieliu** [comment]
Fixed by #33
